packages/syft/tests/syft/core/tensor/autodp/single_entity_phi_test.py

```diff
@@ -17,7 +17,12 @@
 # Global constants
 ishan = Entity(name="Ishan")
 traskmaster = Entity(name="Andrew")
-dims = np.random.randint(10) + 3  # Avoid size 0
+dims = int(max(3, np.random.randint(10) + 3))  # Avoid size 0 and 1
```

Comment:
Strange as to why it is generating tensor of (1,1) at this point

Reply:
Is there any dimensional compression @IshanMi , during SEPT initialization

Reply:
Sometimes, the CI additional randomness, It happened many times, if we run locally  for zillion times it passes, but in CI it fails